brains/andalite.py

```diff
@@ -1,32 +1,38 @@
 import ast
 import asyncio
 import pprint
+import pygame
 import queue
 import threading
 import websockets
 
 from sapiens import Sapiens
 from sync import game_sync_classes
-
+import visualization.textlog
 
 class Andalite(Sapiens):
     name = 'andalite'
     lastSent = {}
 
-    def __init__(self, game, *_, server='ws://localhost:8765', **kwargs):
+    def __init__(self, game, server='ws://localhost:8765', *_, **kwargs):
```

Comment:
wat

Reply:
pycharm was giving me some warning about having named args before a *_

Reply:
it's by design, i'd just disable it :P

Reply:
disappeared into the merge ether